ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -135,10 +135,6 @@ export const WorkflowNodeInputs = (props: {inputs: models.Inputs}) => {
     );
 };
 
-function hasEnv(container: models.kubernetes.Container | models.Sidecar | models.Script): container is models.kubernetes.Container | models.Sidecar {
-    return (container as models.kubernetes.Container | models.Sidecar).env !== undefined;
-}
-
```

Comment:
Are these changes intentional? Perhaps you need to merge with master?

Reply:
My `git merge` messed up. I fixed it